browser/src/canvas/CanvasSectionContainer.ts

```diff
@@ -2113,7 +2113,7 @@ class CanvasSectionContainer {
 			if (options.defer)
 				requestAnimationFrame(this.animate.bind(this));
 			else
-				this.animate(performance.now());
+				this.animate(document.timeline.currentTime as number);
```

Comment:
Can you elaborate the reason for this change?

Reply:
`document.timeline.currentTime` is aligned to the animation timer - if you use `performance.now`, you can end up with a timestamp that's in the future to a subsequent `requestAnimationFrame` (only in Chrome. I assume it fiddles its timestamps a little so animations display more smoothly?)

Reply:
I think you see possible corner cases better than me, after redesigning scroll experience. I don't fully understand the needs for these changes but i am sure that they are harmless. And you saw some danger without them, so we need to get them in :) Thanks for the explanations.